From 0f5b7421d7b56b13f98519218d8b001e81fd802a Mon Sep 17 00:00:00 2001
From: Harini Katakam <harini.katakam@xilinx.com>
Date: Fri, 31 Jan 2020 21:07:21 +0530
Subject: [PATCH 1099/1852] net: macb: Update type of "mac" from char to u8

commit 780956faff2fc6bc1f3332ae9c3c58f2d0ec7397 from
https://github.com/Xilinx/linux-xlnx.git

This is to be compatible with type of ether_addr_copy.
Fixes coverity warning:
(1) Event incompatible_param:
argument of type "const char *" is incompatible with parameter of type
"const u8 *"
(2) Event caretline:
^
4296 			ether_addr_copy(bp->dev->dev_addr, mac);

Signed-off-by: Harini Katakam <harini.katakam@xilinx.com>
Reviewed-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/net/ethernet/cadence/macb_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/cadence/macb_main.c b/drivers/net/ethernet/cadence/macb_main.c
index 4ce60835cfec..3c7122ecc6f8 100644
--- a/drivers/net/ethernet/cadence/macb_main.c
+++ b/drivers/net/ethernet/cadence/macb_main.c
@@ -4326,7 +4326,7 @@ static int macb_probe(struct platform_device *pdev)
 	struct net_device *dev;
 	struct resource *regs;
 	void __iomem *mem;
-	const char *mac;
+	const u8 *mac;
 	struct macb *bp;
 	int err, val;
 
-- 
2.31.1

